ios/sound.h

```diff
@@ -0,0 +1,108 @@
+/******************************************************************************\
+ * Copyright (c) 2004-2020
+ *
+ * Author(s):
+ *  Volker Fischer
```

Comment:
I have not written this file.

Reply:
As I said, it's mainly a copy from macOS.

Reply:
Still, I have not created this file. You have created it. So you should write your name in this file.

Reply:
Ok. I've added my name. But since you wrote most of it for MacOS (at least that's what I assume), I've now written "ann0see based on code from Volker Fischer". I hope that's ok?

Reply:
Yes, that's ok.